### ISSUE TITLE
Packer should fail when creating a floppy file greater than 1.44MB

### ISSUE THREAD

**Issue by icnocop** [hashicorp/packer#4999]
Hi.

I configured `packer.json` with `floppy_files` where the total combined file size of the specified items exceeds 1.44MB.

I'm currently using Packer v0.12.0 on Windows 7 Enterprise 64-bit Service Pack 1.

The packer log did not indicate any warnings or errors.

The floppy file size packer created is 1,440 KB.

For example, I tried to put wget.exe on the floppy, but wget.exe is more than 3MB.

When the operating system tried to run wget.exe as part of the provisioning process (ex. https://github.com/boxcutter/windows/blob/master/floppy/01-install-wget.cmd), it displayed two error messages:
```
[a:\WGET.EXE]
a:\WGET.EXE is not a valid Win32 application.
[OK]
```
```
[cmd.exe - Corrupt File]
The file or directory \wget.exe is corrupt and unreadable. Please run the Chkdisk utility.
[OK]
```

`wget.exe` was the last item in the list of `floppy_files` and the files previously listed do not seem to be corrupt.

`wget.exe` got added to the floppy file but its size was truncated to 1.3MB.

I expected packer to produce an error message if the floppy file exceeds the maximum floppy file size before it gets "truncated".

Thank you.

**Comment by icnocop** [comment]
No, it's not a duplicate.

A floppy file created with a total size of the combination of its contents are less than 1.44MB works without issues.

I think the issue is here:
`https://github.com/hashicorp/packer/blob/81522dced0b25084a824e79efda02483b12dc7cd/common/step_create_floppy.go#L55`

The code doesn't check to see if the size is greater than 1.44MB and just "blindly" truncates it, causing the file(s) that exceed(s) 1.44MB to be corrupt (or maybe not exist).

**Comment by icnocop** [comment]
[Sample packer json and log output](https://gist.github.com/icnocop/28993ef1bf92c08884cae8e5a53197ed)

**Comment by arizvisa** [comment]
I think this was fixed by PR #5802.

PR #5802 should fail critically if it's unable to write a file into the floppy disk's filesystem.
